moshi/src/test/java/com/squareup/moshi/AdapterMethodsTest.java

```diff
@@ -84,136 +86,157 @@
     }
   }
 
-  @Test public void toAndFromWithDelegate() throws Exception {
-    Moshi moshi = new Moshi.Builder()
-        .add(new PointJsonAdapterWithDelegate())
-        .build();
+  @Test
+  public void toAndFromWithDelegate() throws Exception {
+    Moshi moshi = new Moshi.Builder().add(new PointJsonAdapterWithDelegate()).build();
     JsonAdapter<Point> adapter = moshi.adapter(Point.class);
     Point point = new Point(5, 8);
     assertThat(adapter.toJson(point)).isEqualTo("[{\"x\":5,\"y\":8}]");
     assertThat(adapter.fromJson("[{\"x\":5,\"y\":8}]")).isEqualTo(point);
   }
 
-  @Test public void toAndFromWithDelegateWithQualifier() throws Exception {
-    Moshi moshi = new Moshi.Builder()
-        .add(new PointJsonAdapterWithDelegateWithQualifier())
-        .add(new PointWithParensJsonAdapter())
-        .build();
+  @Test
+  public void toAndFromWithDelegateWithQualifier() throws Exception {
+    Moshi moshi =
+        new Moshi.Builder()
+            .add(new PointJsonAdapterWithDelegateWithQualifier())
+            .add(new PointWithParensJsonAdapter())
+            .build();
     JsonAdapter<Point> adapter = moshi.adapter(Point.class, WithParens.class);
     Point point = new Point(5, 8);
     assertThat(adapter.toJson(point)).isEqualTo("[\"(5 8)\"]");
     assertThat(adapter.fromJson("[\"(5 8)\"]")).isEqualTo(point);
   }
 
-  @Test public void toAndFromWithIntermediate() throws Exception {
-    Moshi moshi = new Moshi.Builder().add(new Object() {
-      @FromJson String fromJson(String string) {
-        return string.substring(1, string.length() - 1);
-      }
-
-      @ToJson String toJson(String value) {
-        return "|" + value + "|";
-      }
-    }).build();
+  @Test
+  public void toAndFromWithIntermediate() throws Exception {
+    Moshi moshi =
+        new Moshi.Builder()
+            .add(
+                new Object() {
+                  @FromJson
+                  String fromJson(String string) {
+                    return string.substring(1, string.length() - 1);
+                  }
+
+                  @ToJson
+                  String toJson(String value) {
+                    return "|" + value + "|";
+                  }
+                })
+            .build();
     JsonAdapter<String> adapter = moshi.adapter(String.class);
     assertThat(adapter.toJson("pizza")).isEqualTo("\"|pizza|\"");
     assertThat(adapter.fromJson("\"|pizza|\"")).isEqualTo("pizza");
   }
 
-  @Test public void toAndFromWithIntermediateWithQualifier() throws Exception {
-    Moshi moshi = new Moshi.Builder().add(new Object() {
-      @FromJson @Uppercase String fromJson(@Uppercase String string) {
-        return string.substring(1, string.length() - 1);
-      }
-
-      @ToJson @Uppercase String toJson(@Uppercase String value) {
-        return "|" + value + "|";
-      }
-    }).add(new UppercaseAdapterFactory()).build();
+  @Test
+  public void toAndFromWithIntermediateWithQualifier() throws Exception {
```

Comment:
I think this is strictly worse

Reply:
https://github.com/google/google-java-format/issues/342